### PR TITLE
Activates mice eating wires through a baby event

### PR DESCRIPTION
**Pull request by Time-Green**
Mice will start eating wires after an event is triggered, the event is common, about almost every round.

Earliest start: 10000 ticks (about 16 minutes))
Minimum pop = 30
max_occurences = 1

I was thinking about mice having a 1 in 100 chance to transform into a special electromouse upon eating a wire. If this gets merged, I'll probably do that.

### CODE REVIEW

**Comment by Time-Green** [comment]
I forgot to tell, this is to prevent low-pop station wide power failure within 5 minutes due to mice

**Comment by Cruix** [comment]
:+1:

File: code/modules/events/mousehunger.dm [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/datum/round_event_control/mouse_huner
```
Comment (oisin100) [inline code/modules/events/mousehunger.dm]: Made a typo it seems!

File: code/modules/events/mousehunger.dm [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	name = "Mice hunger for wires"
+	typepath = /datum/round_event/mouse_hunger
+	min_players = 30
+	max_occurrences = 1
+	weight = 20
+	earliest_start = 10000
+
+/datum/round_event/mouse_hunger
+	var/list/mouse_territory = list(/area/maintenance)
```
Comment (oisin100) [inline code/modules/events/mousehunger.dm]: Unused variable! REEEE!

File: code/modules/events/mousehunger.dm [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/datum/round_event_control/mouse_huner
+	name = "Mice hunger for wires"
+	typepath = /datum/round_event/mouse_hunger
+	min_players = 30
+	max_occurrences = 1
+	weight = 20
+	earliest_start = 10000
```
Comment (oisin100) [inline code/modules/events/mousehunger.dm]: Most events if not all that have a custom earliest start value leave a comment letting people know what the earliest start is in minutes.

File: code/modules/events/mousehunger.dm [hunk 4]
```diff
@@ -0,0 +1,18 @@
+/datum/round_event_control/mouse_huner
+	name = "Mice hunger for wires"
```
Comment (oisin100) [inline code/modules/events/mousehunger.dm]: All the words in an event name are usually start with a capital letter.

**Comment by Time-Green** [comment]
Dammit, I forgot to remove some shit with my second commit, I shoudln't have rushed it.
I'll fix it when I can

**Review by X-TheDark — APPROVED** [review]
Moving it to an event is good.
Code also looks good.

**Pull request merged**